From 5a81a8d490e48024d83c5b5ddbc3f7cb59b1ba17 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Wed, 22 Nov 2017 14:28:24 +0200
Subject: [PATCH 089/706] enetc: Update driver version to v0.5

This is the first driver version supporting m0166 simulator model.
New features:
    * Rx/Tx checsum offload
    * RSS hasing support
    * spoof checking and loopback support
    * updated to newer PCI MSIX API (improvement)
Fixes, workarounds, improvements:
	- workaround to allow VLAN traffic (Rx VLAN offload on)
	- various smaller fixes and cleanup patches

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 6037384c0d7ca3a5743fa42961b34bda62fe6df3)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc.c b/drivers/net/ethernet/freescale/enetc/enetc.c
index bf94e3baded7..dd3a4ce3401d 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc.c
@@ -39,7 +39,7 @@
 #include <linux/udp.h>
 
 #define ENETC_DRV_VER_MAJ 0
-#define ENETC_DRV_VER_MIN 4
+#define ENETC_DRV_VER_MIN 5
 
 #define ENETC_DRV_VER_STR __stringify(ENETC_DRV_VER_MAJ) "." \
 			  __stringify(ENETC_DRV_VER_MIN)
-- 
2.17.1

